context/app/static/js/components/Detail/FileTable.jsx

```diff
@@ -70,4 +71,16 @@ function FileTable(props) {
   );
 }
 
+FileTable.propTypes = {
+  files: PropTypes.arrayOf(
+    PropTypes.shape({
+      rel_path: PropTypes.string,
```

Comment:
Here, and with entity_type below, I would lean towards renaming them into camelCase earlier.... Perhaps the line should actually be in the Python? Unless it's useful to flag that these are coming from outside?